src/SourceBuild/tarball/content/repos/arcade.proj

```diff
@@ -2,7 +2,6 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
```

Comment:
I thought the pattern was going to be to always define `BuildCommandArgs` and to default it to `StandardBuildArgs`.  That way, if you need to override any args, you just need to define the override, not change the `BuildCommand` as well.  This was the issue that we saw in msbuild below.

Reply:
I waffled on utilizing that pattern.  The reason I decided against it was because we have so many repos that don't specify any custom args.  Additionally there are a few repos where the custom args should be pushed into the repo itself.  I hope when we are done, custom args will be an edge case.